modules/programs/ion.nix

```
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.ion;
+
+  aliasesStr = concatStringsSep "\n"
+    (mapAttrsToList (k: v: "alias ${k} = ${escapeShellArg v}")
+      cfg.shellAliases);
+in {
+  meta.maintainers = [ maintainers.jo1gi ];
+
+  options.programs.ion = {
+    enable = mkEnableOption "The Ion Shell. Compatible with Redox and Linux.";
```

```suggestion
    enable = mkEnableOption "the Ion Shell. Compatible with Redox and Linux";
```
`mkEnableOption` appends a period automatically.